web/packages/teleport/src/Integrations/Enroll/AwsOidc/AwsOidc.tsx

```diff
@@ -147,6 +152,28 @@ export function AwsOidc() {
           AWS RDS
         </RouteLink>{' '}
         instances during resource enrollment.
+        <P>
```

Comment:
i'm not sure how the ui tests are passing, but can we change this to how i done it here: https://github.com/gravitational/teleport/pull/46700